src/adminsystem.cpp

```diff
@@ -138,23 +159,25 @@ CON_COMMAND_CHAT(ban, "ban a player")
 
 CON_COMMAND_CHAT(mute, "mutes a player")
 {
-	if (!player)
-		return;
+	int iCommandPlayer = -1;
 
-	int iCommandPlayer = player->GetPlayerSlot();
+	if (player)
+	{
+		iCommandPlayer = player->GetPlayerSlot();
 
-	ZEPlayer* pPlayer = g_playerManager->GetPlayer(player->GetPlayerSlot());
+		ZEPlayer *pPlayer = g_playerManager->GetPlayer(iCommandPlayer);
 
-	if (!pPlayer->IsAdminFlagSet(ADMFLAG_BAN))
-	{
-		ClientPrint(player, HUD_PRINTTALK, CHAT_PREFIX"You don't have access to this command.");
-		return;
-	}
+		if (!pPlayer->IsAdminFlagSet(ADMFLAG_BAN))
+		{
+			ClientPrint(player, HUD_PRINTTALK, CHAT_PREFIX "You don't have access to this command.");
+			return;
+		}
 
-	if (args.ArgC() < 3)
-	{
-		ClientPrint(player, HUD_PRINTTALK, CHAT_PREFIX"Usage: !mute <name> <duration/0 (permanent)>");
-		return;
+		if (args.ArgC() < 3)
+		{
+			ClientPrint(player, HUD_PRINTTALK, CHAT_PREFIX "Usage: !mute <name> <duration/0 (permanent)>");
```

Comment:
You sure we don't want this to be shown for the console too? Maybe ClientPrint would print to console for -1 player

Reply:
They should be shown, I just missed them...